app/main.py

```diff
@@ -2,0 +1,42 @@
+from types import TracebackType
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name: str = name
+        self.health: int = health
+        self.hidden: bool = False
+        Animal.alive.append(self)
```

Comment:
In line 11, you are using `self` to access a class attribute. It's a code efficiency issue. Use the class name to get access to the class attributes. So, you should write `Animal.alive.append(self)` instead of `self.alive.append(self)`.